TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/MainActivity.kt

```diff
@@ -42,4 +45,16 @@ class MainActivity : AppCompatActivity() {
                 ).commit()
         }
     }
+
+    /**
+     * Send data from Note Fragment to ItemsListFragment after note contents have been edited
+     *
+     * REVISIT: may just want to encapsulate fields (have to add drawings/photos) in one object
+     *
+     * @param title: updated note title
+     * @param text: updated note text
+     */
+    override fun onNoteUpdate(title: String, text: String) {
```

Comment:
will this work in single screen mode? if not, we may have to consider using metadata files that are saved to the device